src/libsync/propagateuploadng.cpp

```diff
@@ -488,20 +488,12 @@ void PropagateUploadFileNG::slotMoveJobFinished()
         return;
     }
 
-    QByteArray fid = job->reply()->rawHeader("OC-FileID");
-    if(fid.isEmpty()) {
-        qWarning() << "Server did not return a OC-FileID" << _item->_file;
+    if(job->reply()->rawHeader(fileIdHeaderC).isEmpty()) {
```

Comment:
So we assume that (future) servers will continue to always emit both "local" and full IDs?
@PVince81